graphify-out/GRAPH_REPORT.md

```
@@ -35,147 +36,1530 @@
 
 ### Community 3 - "Community 3"
-Cohesion: 0.2
-Nodes (10): DINOv2, Dispersive Loss, InfoNCE, REPA (Representation Alignment), REPA-E, End-to-End LDM Training, Latent Space Collapse, Projection Head (+2 more)
+Cohesion: 0.03
+Nodes (54): PainConfig, resolveReflectionWorkspace(), PainDictionary, shouldIgnorePainProtocolText(), bundlePlugin(), copyRecursive(), appendEvent(), loadImportedHashes() (+46 more)
 
 ### Community 4 - "Community 4"
-Cohesion: 0.2
-Nodes (10): DDO (Direct Discriminative Optimization), Jensen's Inequality, MMDiT (Multimodal Diffusion Transformer), QK-Normalization, Stable Diffusion 3, Implicit Discriminator, Log Likelihood Ratio, Logit-Normal Sampling (+2 more)
+Cohesion: 0.03
+Nodes (72): canArchive(), getAllImplementations(), _handleArchiveImpl(), handleArchiveImplCommand(), _handleListArchivable(), handleBootstrapTools(), scanEnvironment(), getAllCommandDescriptions() (+64 more)
```

markdownlint-cli2 (0.22.0)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)